Paper-MojangAPI/src/main/java/com/destroystokyo/paper/event/brigadier/AsyncPlayerSendCommandsEvent.java

```diff
@@ -28,8 +28,10 @@
  * <p>If your logic is not safe to run asynchronously, only react to the synchronous version.</p>
  *
  * <p>This is a draft/experimental API and is subject to change.</p>
+ * @deprecated For removal, use the new brigadier api.
  */
 @ApiStatus.Experimental
+@Deprecated
```

Comment:
javadocs say for removal, but the annotation doesn't? same for CommandRegisteredEvent